components/dataflow/predict/component.yaml

```diff
@@ -0,0 +1,30 @@
+name: Predict using TF on Dataflow
+description: |
+  Runs TensorFlow prediction on Google Cloud Dataflow
+  Input and output data is in GCS
+inputs:
+  - {name: Data file pattern,   type: {GcsUriPattern: [text, CSV]},      description: 'GCS or local path of test file patterns.'}
```

Comment:
The type is dictionary of key-value property pairs. Can we make this:
type: {GcsUriPattern: {format: text} }?

Reply:
>The type is dictionary of key-value property pairs.
Did we drop type inheritance and generic type support?

>Can we make this:
Yes, we can. Is this the final type structure?

Reply:
Done